### ISSUE TITLE
cannot open 0.17.0 from zip release on mac

### ISSUE THREAD

**Issue by sneak** [ipfs/ipfs-desktop#1922]
- OS: macOS
- Version of IPFS Desktop 0.17.0

```
nostromo:~/Downloads$ shasum IPFS-Desktop-0.17.0-mac.zip
bfb2285198c4172dbe7d54394165163fc9017432  IPFS-Desktop-0.17.0-mac.zip
```

**Describe the bug**

Unzipping the file `IPFS-Desktop-0.17.0-mac.zip` results in an application bundle that cannot be opened.

<img width="532" alt="Screen Shot 2021-10-26 at 03 26 04" src="https://user-images.githubusercontent.com/408977/138860467-ab555ab5-7c28-4d38-8df7-ca244cd1c444.png">

`file` suggests that it's the right arch for this machine:

```
nostromo:~/Downloads/IPFS Desktop.app/Contents/MacOS$ file IPFS\ Desktop
IPFS Desktop: Mach-O 64-bit executable x86_64
```

**Comment by lidel** [comment]
@sneak I don't believe .zip should be used for installation, it may be artifact used for fetching deltas for partial upgrades (or something from older macOS?).

**Comment by chrleon** [comment]
Downloading the DMG works.

**Comment by galargh** [comment]
I do agree dmg is the way to go but, since I had a second I have tried repeating the described steps. Here are some things I've noticed on m1 mac running Monterrey:
- couldn't unzip it with `unzip`; had to fallback to `jar xvf`
- failed to launch after unpack
- briefly opened (the icon showed up) after changing permissions recursively to 755 but crashed with failing to load `'.../IPFS Desktop.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework' (not a mach-o file)` (I don't think electron is significant here - it's just first lexicographically)

**Comment by rigan0707** [comment]
0x7eec6f523a04626dc657e0ec77863d534bdb306c

**Comment by lidel** [comment]
Ok, it seems `zip` is built only to provide interop with something called `Squirrel.Mac` (https://github.com/Squirrel/Squirrel.Mac).
It does not seem to be useful these days –or is it?

 I am going to remove it so mac users are not confused by "broken zip".

**Comment by lidel** [comment]
Reopening, as the removal of this "useless" `.zip` may break autoupdate on macOS:
- https://github.com/electron-userland/electron-builder/issues/2199
  -  rela  apps failing to autoupdate without .zip:  https://github.com/vladimiry/ElectronMail/issues/103, https://github.com/athensresearch/athens/issues/1092#issuecomment-829697980, https://github.com/microsoft/accessibility-insights-web/issues/3942

This is pretty hard  to test, because of apple's walled garden and the notarization security dance.

Next steps:
- [ ] release 0.18.0
- [ ] test autoupdate on macOS by installing 0.17.0 and waiting for it to autoupdate to  0.18.0
  - if it fails, quickly tag 0.18.1 with restored `.zip` for macOS

**Comment by lidel** [comment]
- Yeah, so this is undocumented mess: https://www.electron.build/configuration/mac
  Nowhere it is stated that disabling `zip` will break autoupdate for `dmg` :upside_down_face: 

- I confirmed 0.18.0 fails to update on macOS. 

- I've set staged rollout of 0.18.0 (broken on macOS) to 1%, and will ship 0.18.1 shortly.

**Comment by lidel** [comment]
Shipped fix in 0.18.1 and opened https://github.com/electron-userland/electron-builder/pull/6496 to add electron-builder docs about the bug they were aware of since.. 2017 :|
